### DISCUSSION TITLE
How to apply styles to sub-components

### DISCUSSION THREAD

**Discussion by ondave** [mantinedev/mantine#2376] [Q&A] [answered]
How should sub-components be referenced in the MantineProvider config?
e.g. HoverCard.Dropdown

`<MantineProvider
      theme={{
        components: {
          'HoverCard.Dropdown': {
            styles: { root: { backgroundColor: 'white' } },
          },
        },
      }}
    >`

This does not seem to work?

**Comment by rtivital** [comment, marked as answer]
Sub components do not support styles api on MantineProvider, you need to apply styles to parent component instead:
```tsx
<MantineProvider theme={{ components: { 'HoverCard': { styles: { dropdown: { backgroundColor: 'white' } }, }, }, }} >
```
